src/Features/CSharp/Portable/TypeStyle/UseExplicitTypeCodeFixProvider.cs

```diff
@@ -21,8 +21,10 @@ namespace Microsoft.CodeAnalysis.CSharp.TypeStyle
     [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.UseExplicitType), Shared]
     internal class UseExplicitTypeCodeFixProvider : SyntaxEditorBasedCodeFixProvider
```

Comment:
i'm definitely not a fan of reusing this fixer for that compiler code.  These fixers are really totally intended to be for the actual cases of the IDE analyzers which validate the users's 'var/explicit' type options against the code.  I think it really muddies the waters and makes things less clear when this is used in such a fashion..--

--

To give some perspective, i've put through nearly a dozen PRs just to detangle cases where we had fixers trying to handle too many cases.  It is usually *much* easier to maintain and understand if we keep the different parts simple and clear as to what their responsibility is.